product-service/serverless.yml

```diff
@@ -8,6 +8,9 @@ provider:
   stage: ${opt:stage, 'dev'}
   profile: js-cc4
   region: eu-central-1
+  environment:
+    SERVICE_REGION: ${self:provider.region}
```

Comment:
instead of passing this param you can just get aws region from 'process.env.AWS_REGION'. this env veriable is set by AWS

Reply:
@bogdan-panteleev, nice! thanks for this information! did not know about that. Will update in next pr